www/showtags

```diff
@@ -140,21 +141,28 @@ if (count($tags) == 0) {
         $tu = urlencode($tag);
         $siz = calcDistBin($freq, $freqBins);
         $color = calcDistBin($age, $ageBins);
-
-        echo "<span class='showtags__tag'>"
-            . "<a class=silent href=\"search?searchfor=tag:$tu\">"
-            . "<font size=$siz><span class=\"tagAge$color\">"
-            . "$td</span></font></a></span> ";
+        $currentcolumn = $nextcolumn++;
+        if ($nextcolumn > $totalcolumns) {
+            $nextcolumn = 1;
+        }
+        if ($currentcolumn == 1) {
+            echo "<tr>";
+        }
+        echo "<td>"
+            . "<a href=\"search?searchfor=tag:$tu\">"
+            . "<font size=2>"
+            . "$td</a> ($freq)</font></a></td>";
+        if ($currentcolumn == $totalcolumns) {
+                echo "</tr>";
+            }
     }
-
 }
-
+echo "</table>";
```

Comment:
table should be closed inside the `else` block, one line up, since that's where it was opened.